components/service-operator/apis/storage/v1beta1/s3_types_test.go

```diff
@@ -49,6 +50,28 @@ var _ = Describe("S3Bucket", func() {
 		Expect(o.GetSecretName()).To(Equal("my-target-secret"))
 	})
 
+	It("should base egress whitelisted host name off object name", func() {
+		outputs := cloudformation.Outputs {
+			v1beta1.S3BucketName: "test",
+		}
+
+		ret, err := o.GetServiceEntry(outputs)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(ret.GetObjectMeta().Name).To(Equal(fmt.Sprintf("svcop-s3-%s", o.GetName())))
+		Expect(ret.GetObjectMeta().Namespace).To(Equal(o.GetNamespace()))
+		Expect(ret.GetSpec()["resolution"]).To(Equal("DNS"))
```

Comment:
Could we make use of some better matchers here like `HaveKeyWithValue` so that the errors are easier to grok